pkg/sql/plan/build_test.go

```diff
@@ -767,6 +768,8 @@ func TestDdl(t *testing.T) {
 		"create view v_nation as select n_nationkey,n_name,n_regionkey,n_comment from nation",
 		"CREATE TABLE t1(id INT PRIMARY KEY,name VARCHAR(25),deptId INT,CONSTRAINT fk_t1 FOREIGN KEY(deptId) REFERENCES nation(n_nationkey)) COMMENT='xxxxx'",
 		"create table t2(empno int unsigned,ename varchar(15),job varchar(10) key) cluster by(empno,ename)",
+		"create table t1(a int); lock tables t1 read",
```

Comment:
you should use tpch's table in mock.
this test case not make any sense.  because build_test only test the statements[0], and drop statements[1]。
this sql will test "create table t1(a int)" then finish.

Reply:
I have used it, but get error show can't find table, I will try again